web-app/src/utilities/AxiosInstance.js

```diff
@@ -36,22 +36,27 @@ instance.interceptors.response.use(function(response) {
       data: holidays,
     };
   }
-
   // Append employee to each event (we know its the logged in user) and convert
   // dates to moment objects
   if (response.config.url.includes(`${baseURL}/dashboard/getEmployeeEvents`)) {
-    const events = [...response.data.events];
+    const events = [...response.data];
     const employee = store.getState().USER;
-    for (let event of events) {
-      // Raw dates to moment objects
-      event.start = new moment(event.startDate, 'YYYY-MM-DD');
-      event.end = new moment(event.endDate, 'YYYY-MM-DD');
-      // Append logged in employee
-      event.employee = { ...employee };
-    }
+    const arrayOfEvents = [];
+    events.map(event => {
+      event.events.map(nestedEvent => {
+        const eventId = nestedEvent.eventId ? nestedEvent.eventId : eventId;
```

Comment:
When you're assigning the false `eventId` here in the ternary, does it exist? I can't find anywhere where it's assigned previous to this line.